tests/datasets/test_in_context_learning_datasets.py

```diff
@@ -1168,6 +1168,7 @@ def test_qa_task_evaluation(device, world_size, num_fewshot, dataset_uri, tiny_g
 @pytest.mark.parametrize('generations_per_sample', range(1, 3))
 def test_code_eval_microbatching(device, world_size, num_fewshot, dataset_uri, tmp_path, generations_per_sample):
     pytest.importorskip('datasets')
+    os.environ['CODE_EVAL_DEVICE'] = 'LOCAL'
```

Comment:
```suggestion
def test_code_eval_microbatching(monkeypatch, device, world_size, num_fewshot, dataset_uri, tmp_path, generations_per_sample):
    pytest.importorskip('datasets')
    monkeypatch.setenv('CODE_EVAL_DEVICE', 'LOCAL')
```
Monkeypatch is better since it scopes changes to unit test